src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
-
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+}
```

Comment:
You can get all movie here
```suggestion
type Props = {
  movie: Movie;
}
```